src/content/community/releases/_index.en.md

```diff
@@ -12,6 +12,9 @@ weight = 40
 * The Gateway leader election was enhanced to not restart the pod when leadership is lost to avoid possible data path disruption.
 * Fixed an issue in Service Discovery where stale endpoint IPs, corresponding to services that no longer exist, were returned from DNS
   queries.
+* Sockets from the host are mounted through their parent directory, which ensures that the sockets themselves aren't replaced by directories
+  (which prevents OVN components from starting). Additionally, stray directories are cleaned up at startup. This fixes the known issue with
+  upgrades involving OVN, documented below.
```

Comment:
"...., documented below" without any link can be confusing for the users to follow. 

I hope you are referring to the known issues section. If so, can you include the link to the known issues here?

Reply:
Done, but I couldn’t find a nice way to link to a subsection for a specific version — so if we ever add another “Known issues” sections this will break.